### DISCUSSION TITLE
Questions about LATCH option in BEAMDP

### DISCUSSION THREAD

**Discussion by phantasuson** [nrc-cnrc/EGSnrc#866] [Questions]
Isn't (inclusive filter for bit #25 27) the same as (inclusive filter for region#10)? But here (inclusive filter for bit #25 27) is the same as (inclusive filter for region#9), while (inclusive filter for bit #25 28) is the same as (inclusive filter for region#10)
![image](https://user-images.githubusercontent.com/80371548/163547302-02c6284f-135a-4468-8ab1-80e743b8577b.png)
![image](https://user-images.githubusercontent.com/80371548/163547400-267f8533-c576-4d1b-b360-47e050b32adc.png)
![image](https://user-images.githubusercontent.com/80371548/163547740-d33de107-774a-45ef-9853-5a717370f35c.png)
![image](https://user-images.githubusercontent.com/80371548/163549394-6968d48e-fe2b-466f-8087-0cd537dcaa34.png)

Here is the CM 9 and 10, the phase space file is generated at the back of CM 9
![image](https://user-images.githubusercontent.com/80371548/163549224-8035a747-74d0-47d7-a526-eda5fb6d09bd.png)

**Comment by phantasuson** [comment]
I wander which CM those electrons are coming from

**Comment by rtownson** [comment]
The bits 24-28 (all of these bits together) make up one integer number, which is the region number where a secondary was created (and none are set if it is not a secondary). Thus in BEAMDP it's pretty tricky to filter by region with 24-28, because you would have to figure out the binary representation...

For a bit more description, see section 8 of the BEAMnrc manual.
